harmony/util.py

```diff
@@ -190,6 +196,33 @@ def bool_envvar(name: str, default: bool) -> bool:
         return config
 
 
+def _build_full_user_agent(cfg, service_provider_agt: str) -> str:
+    """
+    Builds a user-agent string that can be passed on to aws or http clients.
+    The user agent may consist of a user agent defined by an env variable passed
+    by newer versions of Harmony, a user agent for this service lib, and an optional user
+    agent that can be provided by users of this lib.
+
+    Parameters
+    ----------
+    cfg : harmony.util.Config
+        The configuration values for this runtime environment.
+    service_provider_agt : string
+        The optional, custom service provider user agent.
+
+    Returns
+    -------
+    string
+        A user agent string.
+    """
+    harmony_agt = cfg.user_agent
```

Comment:
small style nitpick: if it's not absolutely obvious what a word is, we usually spell it out, so `user_agent` and (usually but maybe not universally) `config`

Reply:
Updated that -- thanks for the tip!